File: day6/day6es1.js
function scambia(a, b) {
  let temp = a;
  a = b;
  b = temp;
  console.log(`Dentro la funzione (dopo lo scambio): a=${a}, b=${b}`);
}

let x = 10;
let y = 20;

console.log(`Fuori dalla funzione (prima della chiamata): x=${x}, y=${y}`);
scambia(x, y);
console.log(`Fuori dalla funzione (dopo la chiamata): x=${x}, y=${y}`);

File: day6/day6es1.py
def scambia_valore_py(a, b):
    print("Dentro la funzione (prima dello scambio): a=", a, ", b=", b)
    temp = a       
    a = b          
    b = temp       
    print("Dentro la funzione (dopo lo scambio): a=", a, ", b=", b)

x = 10
y = 20
print("Fuori dalla funzione (prima della chiamata): x=", x, ", y=", y)

scambia_valore_py(x, y)

print("Fuori dalla funzione (dopo la chiamata): x=", x, ", y=", y)
